documentation/src/docs/segment-store-service.md

```diff
@@ -99,25 +99,25 @@ Segment Containers are a logical grouping of Segments, and are responsible for a
 - **Storage Writer**: Processes the durable log operations and applies them to Tier 2 Storage (in the order in which they were received). This component is also the one that coalesces multiple operations together, for better back-end throughput.
 
 ### Segment Container Metadata
-The Segment Container Metadata is critical to the good functioning and synchronization of its components. This metadata is shared across all components and it comes at two levels: "Container-wide metadata" and "per-Segment metadata". Each serves a different purpose and described below.
+The Segment Container Metadata is critical to the good functioning and synchronization of its components. This metadata is shared across all components and it comes at two levels: "Container-wide metadata" and "per-Segment metadata". Each serves a different purpose and is described below.
 
 #### Container Metadata
 
 Each **Segment Container** needs to keep some general-purpose metadata that affects all operations inside the container:
 
-- **Operation Sequence Number**: The largest sequence number assigned by the _Durable Log_. Every time a new operation is received and successfully processed by the _Durable Log_, this number is incremented (its value never decreases or otherwise roll back, even if an operation failed to be persisted).
+- **Operation Sequence Number**: The largest sequence number assigned by the _Durable Log_. Every time a new operation is received and successfully processed by the _Durable Log_, this number is incremented (its value never decrease or otherwise roll back, even if an operation failed to be persisted).
```

Comment:
`its value never decreases` . This is  a singular noun. Revert this change.

Reply:
It should be `its value will never decrease`

Reply:
Accepted suggestion